src/data/set/lattice.lean

```diff
@@ -1501,6 +1501,14 @@ theorem disjoint_image_image {f : β → α} {g : γ → α} {s : set β} {t : s
   (h : ∀ b ∈ s, ∀ c ∈ t, f b ≠ g c) : disjoint (f '' s) (g '' t) :=
 by rintro a ⟨⟨b, hb, eq⟩, c, hc, rfl⟩; exact h b hb c hc eq
 
+lemma disjoint_image_of_injective {f : α → β} (hf : injective f) {s t : set α}
```

Comment:
```suggestion
lemma disjoint.image_of_injective {f : α → β} (hf : injective f) {s t : set α}
```

Reply:
This is in the set namespace so that won't work.

Reply:
```suggestion
lemma _root_.disjoint.set_image_of_injective {f : α → β} (hf : injective f) {s t : set α}
```
I guess